src/TestableIO.System.IO.Abstractions.TestingHelpers/MockFileSystem.cs

```diff
@@ -322,6 +334,16 @@ public void AddDirectory(string path)
                 var s = StringOperations.EndsWith(fixedPath, separator) ? fixedPath : fixedPath + separator;
                 SetEntry(s, new MockDirectoryData());
             }
+
+            lock (drives)
+            {
+                var driveLetter = Path.GetPathRoot(fixedPath);
```

Comment:
Why don't you also use `PathVerifier.NormalizeDriveName` here?

Reply:
Good point, I changed it.